pkg/steps/multi_stage.go

```diff
@@ -371,7 +372,10 @@ func (s *multiStageTestStep) generatePods(steps []api.LiteralTestStep, env []cor
 		delete(pod.Labels, ProwJobIdLabel)
 		pod.Annotations[annotationSaveContainerLogs] = "true"
 		pod.Labels[MultiStageTestLabel] = s.name
+		pod.Spec.ActiveDeadlineSeconds = step.ActiveDeadlineSeconds
```

Comment:
I thought we would not set these?

Reply:
we still set it here in `generatePods`, but then we withhold it later in `multiStageTestStep.runPod`.

Reply:
Obsolete with [the pivot to Vadim's approach][1].

[1]: https://github.com/openshift/ci-tools/pull/1257#issuecomment-705131035